main/tests/server/src/com/google/refine/importers/MarcImporterTests.java

```diff
@@ -104,18 +103,44 @@ public void readMarcFileWithUnicode() throws FileNotFoundException, IOException
+                new String[] { "marc:record - marc:leader", "marc:record - marc:controlfield - tag", "marc:record - marc:controlfield",
+                        "marc:record - marc:datafield - tag", "marc:record - marc:datafield - ind2", "marc:record - marc:datafield - ind1",
+                        "marc:record - marc:datafield - marc:subfield - code", "marc:record - marc:datafield - marc:subfield" },
+                new Serializable[][] {
+                        { "00762cam a22002658a 4500", "001", "93032341", "010", null, null, "a", "93032341" },
+                        { null, "003", "DLC", "020", null, null, "a", "0192814591 :" },
+                        { null, "005", "20000302171755.0", null, null, null, "c", "£4.99" },
+                        { null, "008", "930830s1994    enk           000 0 eng", "040", null, null, "a", "DLC" },
+                        { null, null, null, null, null, null, "c", "DLC" },
+                        { null, null, null, null, null, null, "d", "DLC" },
+                        { null, null, null, "050", "0", "0", "a", "PR2801.A2" },
+                        { null, null, null, null, null, null, "b", "S66 1994" },
+                        { null, null, null, "082", "0", "0", "a", "822.3/3" },
+                        { null, null, null, null, null, null, "2", "20" },
+                        { null, null, null, "100", null, "1", "a", "Shakespeare, William," },
+                        { null, null, null, null, null, null, "d", "1564-1616." },
+                        { null, null, null, "245", "0", "1", "a", "All's well that ends well /" },
+                        { null, null, null, null, null, null, "c", "edited by Susan Snyder." },
+                        { null, null, null, "260", null, null, "a", "Oxford ;" },
+                        { null, null, null, null, null, null, "a", "New York :" },
+                        { null, null, null, null, null, null, "b", "Oxford University Press," },
+                        { null, null, null, null, null, null, "c", "1994." },
+                        { null, null, null, "263", null, null, "a", "9402" },
+                        { null, null, null, "300", null, null, "a", "p. cm." },
+                        { null, null, null, "440", "4", null, "a", "The World's classics" },
+                        { null, null, null, "651", "0", null, "a", "Florence (Italy)" },
+                        { null, null, null, null, null, null, "x", "Drama." },
+                        { null, null, null, "650", "0", null, "a", "Runaway husbands" },
+                        { null, null, null, null, null, null, "x", "Drama." },
+                        { null, null, null, "650", "0", null, "a", "Married women" },
+                        { null, null, null, null, null, null, "x", "Drama." },
+                        { null, null, null, "655", "7", null, "a", "Comedies." },
+                        { null, null, null, null, null, null, "2", "gsafd" },
```

Comment:
Seems like it might be useful to figure out a way to only compared a sub-grid or to be able to specify rows to be skipped or something which would make cases like this less tedious to construct.